keyboards/darkproject/kd87a_bfg_edition/kd87a_bfg_edition.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2022 GSKY <gskyGit@gsky.com.tw>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+
+#define LAYOUT( \
+    k13, k26, k36, k31, k33, k07, k63, k71, k76, ka6, ka7, ka3, ka5,      k97, Ka0, k96, \
+    k16, k17, k27, k37, k47, k46, k56, k57, k67, k77, k87, k86, k66, ka1, kc6, k02, k15, \
```

Comment:
```suggestion
    k16, k17, k27, k37, k47, k46, k56, k57, k67, k77, k87, k86, k66, ka1, kb6, k02, k15, \
```

Reply:
If I remove B11, B13 and replace Kc6 as Kb6, it can compile normally but when comes to Firmware, Keyboard is not recognizing.
And If I only replace Kc6 as kb6, keyboard is recognizing but "insert" key is not working. (where as Kc6 is insert key)